packages/website/src/pages/index.tsx

```diff
@@ -107,9 +108,13 @@ const features: FeatureItem[] = [
 
 function Feature({ title, description }: FeatureItem): JSX.Element {
   return (
-    <div className="col col--12 padding-vert--lg">
-      <h2 className="text--center">{title}</h2>
-      {description}
+    <div className="col col--12 padding-vert--lg ">
+      <div className="text--center">
+        <Heading as="h2" id="title">
```

Comment:
> `id="title"`

There are multiple headings rendered by this component. [Element IDs should be unique within the entire document](https://stackoverflow.com/questions/5611963/can-multiple-different-html-elements-have-the-same-id-if-theyre-different-eleme). Linking to the `#title` hash shouldn't be ambiguous as to which heading it links to.

You'll want the `id` prop here to be computed based on the `title`. Keep in mind that you'll need to replace any spaces with some character like `-`. I'd suggest also lowercasing it. 

A suggestion for your own testing of the PR: make sure you can click the `#` next to the headings. And, when you click that, make sure it doesn't navigate to a _different_ heading in the same page.

Reply:
@JoshuaKGoldberg 
I modified the `title` to `start-type script-eslint-title`. I checked that the tag next to `Why does this project exists?` is moved to `What are ESLint and TypeScript, and how do they compare?`, do you want it not to move?

Reply:
Sorry, I don't follow your explanation here. I tried out the latest version and it's equivalently not working.

But https://github.com/typescript-eslint/typescript-eslint/pull/5951#discussion_r1021995099 should fix things! ✨